docs/SMCWG-charter.md

```diff
@@ -275,9 +275,9 @@ than 50% of the votes cast by the Certificate Consumers must be in favor of the
 ballot. At least one member of each class must vote in favor of a ballot for it
 to be adopted. Quorum is the average number of Member organizations (cumulative,
 regardless of Class) that have participated in the previous three (3) SMCWG
-Meetings or Teleconferences (not counting subcommittee meetings thereof). For
-transition purposes, if three (3) meetings have not yet occurred, quorum is
-three (3).
+Meetings or Teleconferences (not counting subcommittee meetings thereof). No
```

Comment:
From https://github.com/sleevi/cabforum-docs/pull/9/files/7d20e6bcfb580b409a37b8f2f1d0b43d6d10e749:

Quorum rules are simpler if there are no ballots until quorum is determined.  We're unlikely to have a ballot that fast anyway.